### PR TITLE
1) Initialize test structure

### PR DESCRIPTION
**Pull request by magestrio**
## Feature type 
  - [ ] UI
  - [X] Implementation
  - [ ] Specification
  - [ ] CI/CD
  - [ ] Other

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Your code builds clean without any errors or warnings.
- [ ] Tests for changes have been added (for bug fixes / features). If no, why?

### CODE REVIEW

File: packages/zkdb/src/storage-engine/base.ts [hunk 1]
```diff
@@ -108,6 +108,11 @@ export abstract class StorageEngineBase<T, K, M> {
    */
   public abstract readFile(_path: string): Promise<Uint8Array>;
 
+  /**
+   * Clean up the path
+   */
+  public abstract cleanUp(): Promise<boolean>
```
Comment (chiro-hiro) [inline packages/zkdb/src/storage-engine/base.ts:114]: I think `cleanUp()` shouldn't be a part of `MerkleTreeBase` if it was a non-standard method that uses for testing.

Comment (magestrio) [reply packages/zkdb/src/storage-engine/base.ts:114]: `cleanUp()` is a part of the `StorageEngine`. Did you mean `StorageEngine`?

Comment (chiro-hiro) [reply packages/zkdb/src/storage-engine/base.ts:114]: > `cleanUp()` is a part of the `StorageEngine`. Did you mean `StorageEngine`?

Ah! yes. I think we shouldn't implement `cleanUp()` just to make sure it won't be used incorrectly.

Comment (magestrio) [reply packages/zkdb/src/storage-engine/base.ts]: I removed this method. Added the clean up to the test file

**Comment by chiro-hiro** [comment]
@dqtkien Please help me implement github action for test and make sure it work.

**Comment by chiro-hiro** [comment]
@magestrio Please solve the conflict

**Pull request merged**